app/locale/en.js

```diff
@@ -1655,7 +1655,7 @@ module.exports = {
       vp_o_title: 'VP, Operations',
       scd_title: 'Senior Curriculum Developer',
       lcd_title: 'Lead Curriculum Developer',
-      cd_title: 'Curriculum Developer',
+      cd_title: 'Senior Curriculum Developer',
```

Comment:
lol use scd_title instead of replacing this

Reply:
ohohoh sorry, i didn't notice that this is Title name not human name 😂  thanks for find it out